### ISSUE TITLE
sha256sig1 result should be EXTS, not EXTZ

### ISSUE THREAD

**Issue by JamesKenneyImperas** [riscv/riscv-crypto#109]
I think there is a typo on page 37 (sha256sig1) - the result should use EXTS, not EXTZ.

**Comment by ben-marshall** [comment]
You're absolutely right. I honestly thought I'd caught all of those. I'll make sure that gets updated in the next release.
